### ISSUE TITLE
First usage, not pulling all tables

### ISSUE THREAD

**Issue by sej69** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#635]
I'm on the 1 week trial version and am checking to see if this plugin will meet my needs.

I got everything up and running with the tool, it took me a bit to figure out it wasn't going to generate individual .cs files like VS did.  The weird thing is it's only pulling tables that start with "U".

These are my active filter settings:

    FilterSettings.SchemaFilters.Add(new RegexIncludeFilter("dbo.*")); // This includes only dbo schema
    FilterSettings.TableFilters.Add(new RegexExcludeFilter("__MigrationsHistory")); // exclude migration history

So I decided to add in a table manually with this:

    FilterSettings.SchemaFilters.Add(new RegexIncludeFilter("[Cc]ompany.*")); // This includes only dbo schema

However, when I save it all objects under my .tt file go away.  If I comment that line out it imports the tables starting with "U" again.

**Comment by sjh37** [comment]
Hi.
```c#
FilterSettings.SchemaFilters.Add(new RegexIncludeFilter("[Cc]ompany.*")); 
```
Will only include the 'company' schema. Did you actually want that to be a Table filer instead?

**Comment by sej69** [comment]
Ah no…

Ok, then the dbo.* should have been pulling those tables in and it’s not.  How can I troubleshoot this?

**Comment by sjh37** [comment]
Hi Scott,

Change the filter to be
```c#
FilterSettings.TableFilters.Add(new RegexIncludeFilter("[Cc]ompany.*")); 
```
And comment out your schema and other filters. That should hopefully give you what you need.

**Comment by sjh37** [comment]
Hi Scott. Just released v3.2.0 which ups the trail tables & stored procedures 10. I'll close this issue for now, let me know if you need help with filters.
